### DISCUSSION TITLE
Upscaler option didn't appear in re2 non RTX version

### DISCUSSION THREAD

**Discussion by Johard-K-M** [praydog/REFramework#845] [General Questions] [answered]
the mod worked fine in the RTX directxt 12 version, but I changed to direct 11 because of the extra fps, the only thing I did was change the dinput8 file for the version that has TDB in the name, which is the one that works in non-RTX versions , but I can't get the upscaler option to appear, although in the plugin loader tab it appears that the 2 necessary files are loaded, pdfplugin and the dlss file
I tried everything, different versions of the mod, different versions of the DLSS, placing it inside or outside the plugins folder and the option still doesn't come out.

reference image:
https://imgur.com/a/dBzMfuu

**Comment by praydog** [comment, marked as answer]
Upscaler does not work in the old version.

**Reply by Johard-K-M** [reply]
Thanks for the quick reply
